kordac/processors/errors/Error.py

```diff
@@ -0,0 +1,5 @@
+def Error(Exception):
+    """Base class for Errors.
+    (Exceptions from external sources such as inputs).
+    """
+    pass
```

Comment:
yea it was missing one, and then he added it so the little symbol disappeared ;)